File: uri/1012.cpp
#define DESYNC std::ios_base::sync_with_stdio(false); std::cin.tie(0); std::cout.tie(0);
#include <iostream>

int32_t main(void){ DESYNC
    double a,b,c,pi=3.14159;
    std::cin>>a>>b>>c;

    std::cout.precision(3);
    std::cout<<std::fixed;
    std::cout<<"TRIANGULO: "<<(a*c)/2.0<<std::endl;
    std::cout<<"CIRCULO: "<<pi*c*c<<std::endl;
    std::cout<<"TRAPEZIO: "<<(a+b)*c*(1/2.0)<<std::endl;
    std::cout<<"QUADRADO: "<<b*b<<std::endl;
    std::cout<<"RETANGULO: "<<a*b<<std::endl;
}

File: uri/rust/1012.rs
fn get_float_vec() -> Vec<f64> {
    let mut buffer = String::new();
    std::io::stdin().read_line(&mut buffer).expect("Failed");
    let split = buffer.split_whitespace();
    let vec: Vec<&str> = split.collect();
    vec.iter().map(|x| x.trim().parse::<f64>().unwrap()).collect()
}

fn main() {
    let (pi, v): (f64, Vec<f64>) = ( 3.14159, get_float_vec() );

    println!("TRIANGULO: {:.3}", (&v[0] * &v[2]) / 2.0);
    println!("CIRCULO: {:.3}", pi * &v[2].powf(2.0));
    println!("TRAPEZIO: {:.3}", (&v[0]+&v[1]) * &v[2] / 2.0);
    println!("QUADRADO: {:.3}", &v[1].powf(2.0));
    println!("RETANGULO: {:.3}", &v[0] * &v[1]);
}
